src/test/java/com/google/cloud/tools/gradle/appengine/core/DownloadCloudSdkTaskTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2016 Google Inc. All Right Reserved.
```

Comment:
s/2016/2018/

Reply:
Should I change this in every file in the project, or just the ones I've edited since I started? There are quite a few that still say 2016.

Reply:
The copyright date indicates the year that the file was written. Essentially, if someone were to copy the code, they would only violate the copyright if that was done after the copyright year. Therefore, any existing files should keep the original copyright date they were created in.